frontend/src/components/layout/Header.tsx

```diff
@@ -124,6 +124,12 @@ const Header = () => {
     }
   }, [])
 
+  // function toClose() {
```

Comment:
This seems unintentional

Reply:
Yes it is unintentional, I have removed it though thanks for noticing :+1: